Add info about error message enrichment (#661)
akeneo/pim-helpcenter: content/md/connectivity/connection-dashboard.md

You can access the `Monitoring` page by clicking directly on a connection card in the `Connection dashboard` or in the `System` menu, then `Connections`, and on your connection card. 

This page displays information about **the last 100 business errors**. 

Each message contains 3 parts:
- the message: to explain what went wrong during the connection synchronisation,
- internal links: to help you to check the right data inside your PIM,
- external links: to access directly the right pages in our help center.

Moreover, we display some information about the related product: 
- its label and identifier,
- its family,
- and when this is relevant, the attribute channel and locale.

> Finally, on top of the list, you will find a search bar to filter messages. 🔍

Reviewer: ```suggestion
Finally, above the list, you will find a search bar to filter messages. 🔍
```

![Connection monitoring page](../img/connection-monitoring-page.png) 

::: info

> For now, we only track errors on **products creation, update, or deletion**. 

Reviewer: ```suggestion
For now, we only track errors on **product creation, update, or deletion**. 
```
